lib/menu.rb

```diff
@@ -0,0 +1,9 @@
+class Menu
```

Comment:
Like the fact that you have created separate class and spec file for menu as suggested in our learning module.